csrc/scheduler/mma_utils.cpp

```diff
@@ -379,11 +418,6 @@ bool canValidateIsInnerDim(
       if (!split->factor()->isConstInt()) {
         return false;
       }
-      if (split->factor()->evaluateInt() < inner_dim_size) {
-        // This might be too restrictive. Would need more
-        //   bookkeeping to relax.
-        return false;
-      }
```

Comment:
Is this change still needed? I thought this was introduced because we were scheduling the epilogue smem tensor, but now we are just propagating to that tensor, so this is no longer needed?